specs/workspace_spec.rb

```diff
@@ -0,0 +1,120 @@
+require_relative "test_helper"
+
+describe "Workspace" do
+  describe "Initialize" do
+    before do
+      VCR.use_cassette("initialize") do
+        @workspace = SlackBot::Workspace.new
+      end
+    end
+    it "Creates a workspace object" do
+      expect(@workspace).must_be_kind_of SlackBot::Workspace
+    end
+
+    it "Initializes with selected equal to nil" do
+      assert_nil @workspace.selected
```

Comment:
This is a little tricky to Google for but, you can use an `expect` style test with `must_be_nil`.

```suggestion
      expect(@workspace.selected).must_be_nil
```